src/app-layout/__integ__/app-layout-drawers.test.ts

```diff
@@ -184,7 +184,7 @@ for (const visualRefresh of ['true', 'false']) {
     test(
       'updates side split panel position when using different width drawers',
       setupTest(
-        { visualRefresh, splitPanelPosition: 'side', screenSize: { ...viewports.desktop, width: 1500 } },
+        { visualRefresh, splitPanelPosition: 'side', screenSize: { ...viewports.desktop, width: 1450 } },
```

Comment:
Reduce the width, to force split panel to the bottom position, as required by the test.

The refactoring change breakpoint when it happens, so the test needs a change